about.html

```diff
@@ -0,0 +1,184 @@
+          </div>
+        </div>
+      </section>
+      <!-- end sectionone -->
+      <!-- start sectiontwo -->
+      <section class="sectiontwo about center pd-y" id="sectiontwo about">
+        <div class="container about">
+          <h2 class="heading">
+            Unicef's Annual Human Support Conference 2022 Logo
+          </h2>
+          <span class="line"></span>
+          <p>
+            The logo of unicef's annual human support conference was decided
+            through the logo competition from 8.June to 7.July
+          </p>
+          <div class="sectiontwo__container__resposive about">
+            <img
+              src="static/imgs/logo3.png"
+              alt="logo3"
+              class="responsive sectiontwo__container__resposive__respo about"
+            />
+          </div>
+        </div>
+      </section>
+      <!-- end sectiontwo -->
+      <!-- start sectionthree -->
+      <section class="sectionthree about center pd-y" id="sectionthree about">
+        <div class="container">
+          <h3 class="heading">See the past UAHSC global conference</h3>
+          <span class="line"></span>
+          <p>
+            Take a look at the last two UAHSC global conference which take place
+            in USA and in turkey
+          </p>
+          <div class="sectionthree__container__speakers about">
+            <div class="sectionthree__container__speakers__speaker about">
+              <div class="overlay">
+                <h2>2013</h2>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing</p>
+              </div>
+            </div>
+            <div class="sectionthree__container__speakers__speaker about">
+              <div class="overlay">
+                <h2>2013</h2>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- end sectionthree -->
+    </main>
+    <!-- end main -->
+    <!-- start footer -->
+    <footer class="footer" id="footer">
+      <div class="container">
+        <div class="footer__container__parfoot">
+          <div class="footer__container__parfoot__foot">
+            <div class="footer__container__parfoot__foot__responsive">
+              <img
+                src="static/imgs/company-1.png"
+                alt=""
+                class="responsive footer__container__parfoot__foot__responsive__respo"
+              />
+            </div>
+          </div>
+          <div class="footer__container__parfoot__foot">
+            <span
+              >2022 Unicef annual human support in Turkey some Right
+              reserved</span
+            >
+            <p>
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit. Dolore,
+              dolorem! Lorem, ipsum dolor sit amet consectetur adipisicing
+            </p>
+          </div>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] Kindly ensure that you use the correct background color for the footer of the about page so that it matches the Behance design.

**This is how your design looks like:**

![image](https://user-images.githubusercontent.com/86318284/197015514-4d8f229d-146d-48a9-b6d4-b5c097dceb8c.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/197015597-a63a89f2-7aae-4192-bb36-7666d2301e41.png)

Reply:
- Thanks 🙏 for reviewing my pull request I appreciate your review I will work on this part to enhance my code thanks for advising me 👌 💯 
### Thanks again for reviewing my PR  and have a good day😊